src/autoyast-rnc/auth-client.rnc

```diff
@@ -1,301 +0,0 @@
-default namespace = "http://www.suse.com/1.0/yast2ns"
```

Comment:
just be sure you do not break old autoyast profiles. Customers are very sensitive to such changes(often use same profiles for several distributions, own hand-crafted profiles, etc.). Usually we have rnc to define some kind of interface that is used in autoyast profiles. So it is not mandatory, but it helps customers to validate, that their profile match our expectation.